00-basics/002-javascript/freeCodeCamp_03_17.js

```diff
@@ -0,0 +1,218 @@
+function myStr() {
+      let someAdjective = "awesome!";
+     let myStr = "Learning to code is ";
+    myStr += someAdjective;
+    console.log(myStr);
+}
+myStr();
+
+function lastNameLength() {
+     lastNameLength = 0;
+    let lastName = "Lovelace";
+    lastNameLength = lastName.length;
+    console.log(lastNameLength);
+}
+lastNameLength();
+
+function firstLetter() {
+    let firstLetterOfLastName = "";
+    let lastName = "Lovelace";
+    firstLetterOfLastName = lastName[0];
+    console.log(firstLetterOfLastName);
+}
+firstLetter();
+
+function undStr () {
+     myStr = "Jello World";
+    myStr = "Hello World!";
+    console.log(myStr);
+}
+undStr();
+
+function nthCharacter() {
```

Comment:
technically speaking, current implementation is `thirdCharacter`. To make it `nthCharacter` it would have to be `function nthCharacter(letterIndex)`